### DISCUSSION TITLE
need a longtime url of these dailymotion streams

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#152] [Q&A]
can someone help me for extracting the link of the makan33 channel from israel *
https://www.makan.org.il/live/tv.aspx?stationid=25
and kan11 in hebreu
https://www.kan.org.il/live/tv.aspx?stationid=2
and kan kids hebreu
https://www.kankids.org.il/live/tv.aspx?stationid=20
and also a link for i24 news arabic
https://www.i24news.tv/ar/%D9%85%D8%A8%D8%A7%D8%B4%D8%B1

**Comment by BellezaEmporium** [comment]
1) http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/embed/video/k2slq6Tpsh2bm4wA43P?autoplay=1

**Reply by ARAB-IPTV** [reply]
> 1. http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/embed/video/k2slq6Tpsh2bm4wA43P?autoplay=1

thank you , what about the other channels !

**Reply by BellezaEmporium** [reply]
2) http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7wjmog
3) http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/embed/video/k55PfgB8EU6SpzwA46D geoblocked
4) http://free.fullspeed.tv/iptv-query?streaming-ip=https://players.brightcove.net/5377161796001/oHEyVVUhv_default/index.html?videoId=6168209415001

**Reply by ARAB-IPTV** [reply]
thank you s o much bro ismael 👍

**Comment by Sphinxroot** [comment]
i24 news arabic

https://cdn.i24news.tv/upload/image/1352b9caa849f9325c12157cf9dfc47bc0e3c73e.png

Adaptative
https://bcovlive-a.akamaihd.net/846c532fd891444e945b2ab6b2f766ec/eu-central-1/5377161796001/playlist.m3u8
1080p
https://bcovlive-a.akamaihd.net/846c532fd891444e945b2ab6b2f766ec/eu-central-1/5377161796001/profile_0/chunklist.m3u8
720p
https://bcovlive-a.akamaihd.net/846c532fd891444e945b2ab6b2f766ec/eu-central-1/5377161796001/profile_1/chunklist.m3u8
540p
https://bcovlive-a.akamaihd.net/846c532fd891444e945b2ab6b2f766ec/eu-central-1/5377161796001/profile_2/chunklist.m3u8
360p
https://bcovlive-a.akamaihd.net/846c532fd891444e945b2ab6b2f766ec/eu-central-1/5377161796001/profile_3/chunklist.m3u8
